app/controllers/users_controller.rb

```diff
@@ -303,4 +303,13 @@ def followers
   def set_user
     @user = User.find_by(username: params[:id])
   end
+  private
+    def user_params
+        params.require(:user).permit(:username, :email, :password, :password_confirmation, :openid_identifier, :key, :photo, :photo_file_name, :bio, :status)
+    end
+	def spamaway_params
```

Comment:
Small indentation fix - can we make these even with above lines? Thanks!

Reply:
One of the reasons I wanted us to have Rubocop on CI....